### ISSUE TITLE
help a poor macer!

### ISSUE THREAD

**Issue by LawKKK** [RoanH/KeysPerSecond#74]
it's probably because I didn't go through all the descriptions carefully, but I'm enable to figure out how to download the ware on Mac, could you plz offer some help?

**Comment by RoanH** [comment]
Yeah, the .jar file is the correct one. As long as Java is installed double clicking it or maybe some form of open with Java should work, though I can't offer any more specific advice.

**Comment by RoanH** [comment]
Going to close this for now, feel free to reopen if you things are still unclear.
